lib/config.js

```diff
@@ -40,7 +49,128 @@ module.exports.getConfig = async function getConfig({
     config.replacers = []
   }
 
-  config['sort-direction'] = validateSortDirection(config['sort-direction'])
+  if (error) throw error
 
   return config
 }
+
+const category = Joi.object()
+  .keys({
+    title: Joi.string().required(),
+    label: Joi.array()
```

Comment:
Currently, `label` can also be a `string`, does this account for that?

Reply:
the `.single()` accounts for that